index.html

```diff
@@ -258,6 +262,60 @@ <h3>Video Content Hints</h3>
       </p></td></tr>
     </tbody></table>
   </section>
+  <section>
+    <h2>Behaviors of other components based on content-hint</h2>
+    <section>
+      <h3>Behavior of a MediaStreamTrack</h3>
+      <p>A MediaStreamTrack for which a contentHint attribute has been
+      set MUST set the following default settings for constrainable
+        values:</p>
+        <ul>
+          <li>For an audio track with the value "music",
+            echoCancellation, autoGainControl and noiseSuppression
+            default to "false".</li>
+          <li>For an audio track with the value "speech",
+            echoCancellation and autoGainControl default to
+            "true".</li>
+        </ul>
+        <p>There is no change to the MediaTrakcCapabilities based on
+          this value; constraints may still cause these settings to
+          have any legal value.
```

Comment:
We're adding direct controls for the default (unconstrained) values of constrainable properties, so I think we have to spell out interactions between constraints and these controls a bit more, since both can change over time.

Since they're all booleans, it might suffice to say something like: *"The default value MUST be chosen if it satisfies the constraints."*

We should probably also specify when they must affect the sound, e.g with steps in the setter:

 1. For each constrainable property of echoCancellation, autoGainControl and noiseSuppression, if newDefault satisfies applied constraints, set the constrainable property's current setting value to newDefault.
 2. In parallel update audio processing to respect the new settings.